interface/patient_file/addr_label.php

```diff
@@ -5,7 +5,7 @@
 * Program for displaying Address Labels 
 * via the popups on the left nav screen
 * 
-* Copyright (C) 2014 Terry Hill <terry@lillysystems.com> 
+* Copyright (C) 2014-2017 Terry Hill <teryhill@librehealth.io> 
```

Comment:
First publish date is fine by (c) law.  No actual need for expanded, unless you just want to so that dates of re-publish are clear.